packages/common/src/PublicNetworks.js

```diff
@@ -43,9 +43,23 @@ const PublicNetworks = {
     daiAddress: "0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99",
     wethAddress: "0xd0A1E359811322d97991E03f863a0C30C2cF029C"
   },
+  137: {
+    name: "polygon-matic",
+    etherscan: "https://explorer-mainnet.maticvigil.com/",
+    customTruffleConfig: {
+      confirmations: 2,
+      timeoutBlocks: 200,
+      gasPrice: 1000000000 // 1 gWei
+    }
+  },
   80001: {
-    name: "mumbai",
-    etherscan: "https://explorer-mumbai.maticvigil.com/"
+    name: "polygon-mumbai",
+    etherscan: "https://explorer-mumbai.maticvigil.com/",
+    customTruffleConfig: {
+      confirmations: 2,
+      timeoutBlocks: 200,
+      gasPrice: 1000000000 // 1 gWei
```

Comment:
minor nit: have you thought about just importing web3 at the top and using it to get 1 GWEI so this is easier to read (we do this in the truffle config to interpret the user input).

```js
const Web3 = require("web3");

...
gasPrice: Web3.utils.toWei("1", "gwei");
```